packages/launchpad/src/setup/OpenBrowser.vue

```diff
@@ -40,6 +40,18 @@ query OpenBrowser {
   }
   ...WarningList
   currentTestingType
+  versions {
+    current {
+      id
+      version
+      released
+    }
+    latest {
+      id
+      version
+      released
+    }
+  }
```

Comment:
Are these additions to the query necessary? I would have expected them to be used in this component (or moved to a fragment?)

Reply:
Yeah I wasn't really happy with this. I wanted to land for sure on where we wanted to trigger the additional telemetry to go through. Updated here:

[`19b094d` (#20107)](https://github.com/cypress-io/cypress/pull/20107/commits/19b094d70148ddc58e68b5c224f33e940137306a)

[`ae508f1` (#20107)](https://github.com/cypress-io/cypress/pull/20107/commits/ae508f18262baeba7bd1a4c37b8dd93a9df804f5)